lib/openstack_query/handlers/server_side_handler.py

```diff
@@ -26,6 +26,12 @@ class ServerSideHandler(HandlerBase):
     def __init__(self, kwarg_mappings: ServerSideFilterMappings):
         self._server_side_filter_mappings = kwarg_mappings
 
+    def get_supported_props(self, preset: QueryPresetsGeneric) -> List[PropEnum]:
+        """
+        Gets a list of all supported properties for a given preset
+        """
+        return list(self._server_side_filter_mappings[preset].keys())
```

Comment:
I didn't manage to write a test for this, we should add one

In this case I think a really simple stub would work to just check we get the top level keys from a nested dict of NonCallableMocks

Reply:
there is a test for this